README.md

```diff
@@ -26,11 +26,11 @@ available on some systems or can be installed with `spack` (see
 https://spack.readthedocs.io/en/latest/getting_started.html):
 
 ```
-spack install cabana@master+cajita+silo
+spack install cabana@master+grid+hdf5
 ```
 
 Alternatively, Kokkos can be built locally, followed by Cabana:
-https://github.com/ECP-copa/Cabana/wiki/1-Build-Instructions
+https://github.com/ECP-CoPA/Cabana/wiki/1-Build-Instructions
```

Comment:
When I use 

"https://github.com/ECP-CoPA/Cabana/wiki/1-Build-Instructions" 

it routes me to 

"https://github.com/ECP-copa/Cabana/wiki/1-Build-Instructions"

Reply:
Yes, capitalization doesn't matter